source/_integrations/ecoforest.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Ecoforest
+description: Instructions on how to integrate Ecoforest fireplaces with Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2023.10
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@pjanuario'
+ha_domain: ecoforest
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The Ecoforest integration allows monitoring and control of local [Ecoforest](https://ecoforest.com) fireplaces in Home Assistant. These fireplaces are manufactured in Spain.
+
+There is currently support for the following device platforms within Home Assistant:
+
+- [Sensor](#sensor)
+
+{% include integrations/config_flow.md %}
+
+<div class='note'>
+Devices must first be connected to Wi-Fi or LAN using the app or website provided by the device manufacturer. It is not possible to connect devices to Wi-Fi using this integration.
+</div>
+
+### Credentials
+
+To configure the Ecoforest integration you will need to enter your Ecoforest credentials which are the same ones you would use with the manufacturer app. The image below shows how to obtain the credentials:
+
+- Username: use the serial number of the device as identified by 1.
+- Password: use the first 8 characters of the wifi password as identified by 4.
+
+![Ecoforest Credentials](/images/integrations/ecoforest/credentials.png)
+
```

Comment:
@joostlek @pjanuario It doesn't make much sense to have install instructions above how to get the details needed to install. This is a prerequisite...

Reply:
I was thinking in the UX for less technical users, I can remove if we fill it shouldn't be there.